scripts/info.coffee

```diff
@@ -13,7 +13,8 @@ moment = require 'moment'
 module.exports = (robot) ->
   robot.respond /(info) (.+)$/i, (msg)  ->
     query = msg.match[2].toLowerCase()
-    robot.http(process.env.INFO_SPREADSHEET_URL)
+    if query.length > 3
```

Comment:
@aseem09 I like the effort to improve the functionality of the bot. But this and the other commit are not logically related. If you want to add this `spam` filter then please submit it as an independent PR.